src/UI/Settings.js

```diff
@@ -1100,7 +1133,11 @@ function Settings(props) {
         </Form>
         <Form onSubmit={handleSubmit}>
           <H3>Change border</H3>
-          <Input placeholder="5px solid #ff0000 or string" ref={borderInput} />
+          <Input
+            defaultValue={theme.border}
```

Comment:
You should not rely on the theme, instead rely on the useTheme() object that has access to all the theme attributes like this:
useTheme().primary_color
You have multiple examples in this file to see how it's used.